src/main/java/TaskList.java

```diff
@@ -0,0 +1,63 @@
+public class TaskList {
+    private Task[] taskList;
+    private int taskCount;
+
+    public TaskList() {
+        taskList = new Task[100];
+        taskCount = 0;
+    }
+
+    // Adds task to taskList using the Command object
+    public String addTask(Command command) {
+        // First argument will always be the description of the task
+        String taskName = command.getArgument(0).trim();
+        // Extra argument for Deadline and Event
+        String extraArg;
+
+        switch(command.getCommand().toLowerCase()) {
+        case "todo":
+            taskList[taskCount] = new Todo(taskName);
+            break;
+        case "deadline":
+            extraArg = command.getArgument(1).substring(3); // removing the "by "
+            taskList[taskCount] = new Deadline(taskName, extraArg);
+            break;
+        case "event":
+            extraArg = command.getArgument(1).substring(3); // removing the "at "
+            taskList[taskCount] = new Event(taskName, extraArg);
+        }
+        taskCount++;
+        return "Added: " + taskName;
+    }
+
+    // Returns a string indicating completion of the task.
+    // If it's already complete, it will say so by returning the right sentence.
+    public String[] completeTask(String taskIDString) {
+        if (taskIDString == null) {
+            return new String[]{"You need to specify a task number!","Use list to check."};
+        }
+
+        int taskID = Integer.parseInt(taskIDString);
+        if (taskList[taskID-1].getDone()) {
+            return new String[]{"This task is already complete!", "Did you perhaps mean another task?"};
+        } else {
+            taskList[taskID-1].setDone();
+            return new String[]{"I've marked this task as done:", "[\u2713] " + taskList[taskID-1].getName()};
+        }
+    }
+
+    // Returns a string array of all the tasks to be done
+    public String[] listTasks() {
+        String[] outputList = new String[taskCount + 1];
+        outputList[0] = "Here are the tasks in your list:";
+
+        Task eachTask;
```

Comment:
Overall, very clear naming on both variables and methods all aligning to pascal case and camel case respectively. They are also named very clearly to tell what they represent!